File: util/nums.py
import glob, re

data = list()
files = glob.glob("../_posts/*.md")
for file_name in files:
  with open(file_name, "r", encoding="UTF-8") as f:
    text = f.read()
    result = re.search(r"num: (\d+)", text)
    if result is None:
      continue
    data.append(int(result.group(1)))

print(sorted(data))

File: util/nums.rb
require "digest"

Encoding.default_external = 'utf-8'

data = []
Dir.glob("../_posts/*.md") do |url|
  text = File.read(url)
  text.scan(/num: (\d+)/) do |s|
    data << $1.to_i
  end
end

data.sort!

puts data

